lib/aurora_bootstrapper/notifier.rb

```diff
@@ -12,7 +12,48 @@ def export_date
     
     # ENV is string to string dictionary
     def export_date_override
-      DateTime.now.strftime("%Y-%m-%d") if ENV.key?('EXPORT_DATE_OVERRIDE')
+      datetime = nil
```

Comment:
a couple things here:
1. this method is pretty involved an difficult to test
2. it has no tests. 

Let's break this up into a few smaller pieces and add tests